src/legacy/core_plugins/kibana/public/dashboard/index.ts

```diff
@@ -37,14 +31,8 @@ import './dashboard_config';
 async function getAngularDependencies(): Promise<LegacyAngularInjectedDependencies> {
   const injector = await legacyChrome.dangerouslyGetActiveInjector();
 
-  const Private = injector.get<IPrivate>('Private');
-
-  const savedObjectRegistry = Private(SavedObjectRegistryProvider);
-
   return {
     dashboardConfig: injector.get('dashboardConfig'),
```

Comment:
🎉Down to one, nice! We should probably just move that flag into `kibana_legacy`, then dashboard is almost ready to go